translations/french/helptickets.ts

```diff
@@ -0,0 +1,166 @@
+		"Do not PM staff if you are locked (signified by the symbol <code>‽</code> in front of your username). Locks are a different type of punishment; to appeal a lock, make a help ticket by clicking the Back button and then selecting the most relevant option.": "N'envoie pas de messages privés au membres du staff d'une room si tu es lock (indiqué par le symbole <code>‽</code> devant ton pseudo). Les lock sont des sanctions différentes ; pour contester un lock, ouvre un help ticket en cliquant sur le bouton Retour, et en selectionnant l'option appropriée.",
+		"Maybe one of these options will be helpful?": "Un de ces boutons pourra peut-être t'aider ?",
+		"If you lost your password, click the button below to request a password reset. We will need to clarify a few pieces of information before resetting the account. Please note that password resets are low priority and may take a while; we recommend using a new account while waiting.": "Si tu as oublié ton mot de passe, clique sur le bouton ci-dessous pour demander à ce qu'il soit réinitialisé. Avant de réinitialiser ton mot de passe, sache que les réinitialisations de mot de passe sont ne sont pas prioritaires et peuvent prendre un moment ; nous recommendons à ce que tu utilises un nouveau pseudo en attendant.",
+		"Request a password reset": "Demande une réinitialisation de mot de passe",
+		"If you are a room driver or up in a public room, and you need help watching the chat, one or more global staff members would be happy to assist you!": "Si tu es room driver ou plus haut dans une room publique et que tu as besoin d'aide pour surveiller la room, un ou plusieurs membres du global staff seraient heureux de t'aider !",
+		"If your issue is not handled above, click the button below to talk to a global staff member. Please be ready to explain the situation.": "Si le problème rencontré n'est pas pris en compte par un des cas ci-dessus, clique sur le bouton ci-dessous pour discuter avec un membre du global staff. Merci de bien expliquer la situation.",
+		"Malformed help request.": "Requête mal formulée.",
+		"PM Harassment": "Harcèlement en messages privés",
+		"Battle Harassment": "Harcèlement en match",
+		"Inappropriate Username": "Pseudo inapproprié",
+		"Inappropriate Pokemon Nicknames": "Surnom de Pokémon inapproprié",
+		"Appeal": "Appel",
+		"IP-Appeal": "Appel IP",
+		"ISP-Appeal": "NOT SURE TBH LOL",
```

Comment:
Please don't put this in the final translation. It's better to leave something in English than to totally obscure its meaning.

Reply:
My bad, I didn't notice this when scanning, I'll let the person who did the translation to know for future reference though

Reply:
Apparently he wants to review the rest of the translations too so I'll fix this along with any others he decides to alter